src/components/[guild]/Requirements/components/DataProviderRequirement.tsx

```diff
@@ -50,6 +57,9 @@ const DataProviderRequirement = ({
           <Text
             fontWeight={"medium"}
             sx={{ fontSize: "sm", "& *": { fontSize: "inherit" } }}
+            whiteSpace={"nowrap"}
```

Comment:
These three props could just be `noOfLines={1}`

Reply:
niiiiiice 🔥